pkg/webhooks/mutator/hyperConvergedMutator.go

```diff
@@ -106,20 +107,42 @@ func (hcm *HyperConvergedMutator) mutateHyperConverged(_ context.Context, req ad
 	return admission.Allowed("")
 }
 
-func mutateEvictionStrategy(hc *hcov1beta1.HyperConverged, patches []jsonpatch.JsonPatchOperation) []jsonpatch.JsonPatchOperation {
+func mutateEvictionStrategy(ctx context.Context, cli client.Client, hc *hcov1beta1.HyperConverged, patches []jsonpatch.JsonPatchOperation) []jsonpatch.JsonPatchOperation {
 	if hc.Status.InfrastructureHighlyAvailable == nil || hc.Spec.EvictionStrategy != nil { // New HyperConverged CR
 		return patches
 	}
 
-	var value = kubevirtcorev1.EvictionStrategyNone
-	if *hc.Status.InfrastructureHighlyAvailable {
-		value = kubevirtcorev1.EvictionStrategyLiveMigrate
+	workerNodes := &corev1.NodeList{}
+	err := cli.List(ctx, workerNodes, client.MatchingLabels{"node-role.kubernetes.io/worker": ""})
```

Comment:
not sure this label is enough. I case the workloads node-placement is set, I think we will need to use it instead.

@orenc1 - WDYT?